sources/packages/web/src/store/index.ts

```diff
@@ -1,14 +1,15 @@
 import { createStore, StoreOptions } from "vuex";
 import { RootState } from "./states";
 import { auth } from "./modules/auth/auth";
+import { student } from "./modules/student/student";
 
 const storeOptions: StoreOptions<RootState> = {
   state: {
     version: "1.0.0"
   },
   modules: {
-    auth
-  }
+      auth,student
```

Comment:
Please, considerer breaks the line or add a space before "student".

Reply:
removed auth related files and mentions